big-practice/src/js/pages/login.page.js

```diff
@@ -0,0 +1,100 @@
+import { formValidate } from '../validates/form.validate';
+import { apiService } from '../services/api.service';
+import { DocumentHelper } from '../helpers/show-message.helper';
+import { ACCOUNTS_API } from '../constants/url-api.constant';
+import { MESSAGES, EMPTY_TEXT } from '../constants/message.constant';
+import { DASHBOARD_PAGE, USERNAME_KEY } from '../constants/app.constant';
+
+class Login {
+  constructor() {
+    // Query elements
+    this.emailInput = document.getElementById('email');
```

Comment:
do we need to declare these variables in constructor?

Reply:
updated!